src/components/tabs/tabs.test.js

```diff
@@ -78,6 +78,27 @@ describe('/components/tabs', () => {
         })
         expect(secondTabPanelIsHidden).toBeFalsy()
       })
+
+      describe('when the tab contains a DOM element', () => {
+        it('should display the tab panel associated with the selected tab', async () => {
+          await page.goto(baseUrl + '/components/tabs/preview', { waitUntil: 'load' })
+
+          await page.evaluate(() => {
+            // Replace contents of second tab with a DOM element
+            const secondTab = document.body.querySelector('.govuk-tabs__list-item:nth-child(2) .govuk-tabs__tab')
+            secondTab.innerHTML = '<span>Tab 2</span>'
+          })
+
+          // Click the second tab
+          await page.click('.govuk-tabs__list-item:nth-child(2) .govuk-tabs__tab')
```

Comment:
Should the click event be on the new `<span>` element?

Reply:
…jinx? 😆

Reply:
Okay I've added `span` to the selector. As the span covers the whole label, the test was actually correctly failing without the bubbling fix as the span was always clicked on - whereas adding more text inside the label made the test pass as the span wasn't clicked on anymore. But agree that it's more correct / explicit clicking on the span.